docs/blog/2019-01-25-mesh-loadtest.md

```diff
@@ -0,0 +1,82 @@
+# Performance Testing of Service Mesh Receiver
+
+- Auther: Hongtao Gao, Apache SkyWalking & ShardingShpere PMC
+- [GitHub](https://github.com/hanahmily), [Twitter](https://twitter.com/hanahmily), [Linkedin](https://www.linkedin.com/in/gao-hongtao-47b835168/)
+
+Jan. 25th, 2019
+
+Service mesh receiver was first introduced in Apache SkyWalking 6.0.0-beta. It is designed to provide a common entrance for receiving telemetry data from service mesh framework, for instance, Istio, Linkerd, etc. What’s the service mesh? According to Istio’s explain:
+
+The term service mesh is used to describe the network of microservices that make up such applications and the interactions between them.
+
+As a PMC of Apache SkyWalking, I tested trace receiver and well understood the performance of collectors in trace scenario. I also would like to figure out the performance of service mesh receiver.
```

Comment:
PMC=Project Management Committee. So you should be PMC member.

Reply:
Fixed this.